scripts/performance/README.md

```diff
@@ -0,0 +1,81 @@
+# Performance Scripts
+
+## Overview
+This directory contains scripts used for monitoring the performance of the EdgeX services. The scripts are written in Shell.
+
+## Structure
+- `ram_cpu.sh`: Script to monitor RAM and CPU usage of specific EdgeX services.
+- `transaction_bandwidth.sh`: Script to record the number of messages and total bytes received per second.
+- `transaction_latency.sh`: Script to record the latency of each message transmission from southbound to northbound.
```

Comment:
Since you added scripts for darwin, shall we also move these scripts into debian directory? You can also mention that STM32MP157 is installed with debain-like OS, so choose scripts under debian.